dotCMS/src/integration-test/java/com/dotcms/util/AnnotationUtilsTest.java

```diff
@@ -0,0 +1,69 @@
+package com.dotcms.util;
+
+import com.dotcms.util.annotationtest.TestActionlet1;
+import com.dotcms.util.annotationtest.TestActionlet2;
+import com.dotcms.util.annotationtest.TestJsonSerializer;
+import com.dotcms.util.annotationtest.TestResource;
+import com.dotmarketing.portlets.workflows.actionlet.Actionlet;
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+import javax.ws.rs.Path;
+import java.util.Set;
+
+public class AnnotationUtilsTest {
+
+    @BeforeClass
+    public static void prepare() throws Exception {
+        //Setting web app environment
+        IntegrationTestInitService.getInstance().init();
+    }
+
+    @Test
+    public void test_scanClassAnnotatedBy_Actionlet_found(){
+
+        final Set<Class<?>> annotatedClasses = AnnotationUtils.scanClassAnnotatedBy("com.dotcms.util.annotationtest", Actionlet.class);
```

Comment:
Codacy found an issue: [The String literal "com.dotcms.util.annotationtest" appears 4 times in this file; the first occurrence is on line 28](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=4888160)